taskiq-redis/taskiq/result_backend/redis_backend.py

```diff
@@ -0,0 +1,92 @@
+import pickle
+from typing import Any, Dict, TypeVar
+
+from redis_client import RedisClient  # type: ignore
```

Comment:
I have problem with mypy, becouse it can't find `implementation or library stub` for my files. I'll try to solve this shit later

Reply:
That shit is unsolvable. 

Just add this in your `pyproject.toml`

```toml
[tool.mypy]
strict = true
ignore_missing_imports = true
allow_subclassing_any = true
allow_untyped_calls = true
pretty = true
show_error_codes = true
implicit_reexport = true
allow_untyped_decorators = true
warn_return_any = false

[[tool.mypy.overrides]]
module = [
    'redis.asyncio'
]
ignore_missing_imports = true

[tool.isort]
profile = "black"
multi_line_output = 3
```

This thing will stop mypy for asking you to add types-redis and will make mypy checks more strict. Also it configures isort to match black's style formatting.

Reply:
Done. Thank you)